src/app/shared/form-field/form-field.ts

```diff
@@ -17,7 +18,8 @@ export class TextFormField extends FormField {
        public label: string, 
        public defaultValue: string = '',
        public placeholder: string = '',
-       public validations: Validator[] = []) {
+       public validations: Validator[] = [],
+       public tooltip: Tooltip = new Tooltip()) {
```

Comment:
Then it won't be of type `Tooltip` i think. 'd be defined as type `any` and explicitly will need to handle if tooltip not available in field component's view.